go/client/cmd_login.go

```diff
@@ -79,7 +79,7 @@ func (c *CmdLogin) Run() error {
 
 	err = client.Login(ctx,
 		keybase1.LoginArg{
```

Comment:
in my mind this is more a side effect of how this is handled in go. from my perspective if something is optional then it should be optional in the avdl and we should be ok to not send it. i guess a more ideal scenario is a wrapper that hides this on the go side more on the rpc side where it'll automatically make it a string and do the checking once?